ios/fastlane/Fastfile

```diff
@@ -174,8 +174,7 @@ platform :ios do
     builder.build_app_store(
       Constants.SCHEME_NAME_PRODUCTION,
       Constants.PRODUCT_NAME_PRODUCTION,
-      Constants.BUNDLE_ID_PRODUCTION,
-      true
```

Comment:
Can not we set it to false here instead?

Reply:
From Xcode 14, `bitcode=true` is deprecated, so I prefer to remove the parameter `include_bitcode`

Reply:
You can check more details here: https://developer.apple.com/documentation/xcode-release-notes/xcode-14-release-notes

![Screenshot 2023-01-31 at 14 29 51](https://user-images.githubusercontent.com/19943832/215694602-ee2a6a27-6448-45c9-9ac5-77047f450c73.png)

Reply:
@suho Thanks for your insight 👍